benchmarks/nemo/nemolite2d/manual_versions/regent/Makefile

```diff
@@ -0,0 +1,16 @@
+F90 = gfortran
```

Comment:
I'm not sure at this point if it would support other compilers (as it relies on libgfortran). I'd rather just add to the comments that only that works for now.

Reply:
That's a good point. So then, can you write that gfortran is a requirement due to importing libgfortran during the execution in the top-level comment for documentation.

And you will need to remove the F90FLAG that I suggested in the previous review (oops) as this could contain flags for other compilers.

Reply:
Done